gbpservice/nfp/service_vendor_agents/vyos/oc_config_server/server2.py

```diff
@@ -460,6 +461,18 @@ def del_rule():
         return json.dumps(dict(status=True))
 
 
+@app.route('/configure-rsyslog-as-client', methods=['POST'])
+def configure_rsyslog_as_client():
```

Comment:
We need to check about the uniformity of the API names when compared with existing REST calls.